src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
this code could be optimized and written more readable. we can use just 2 filters in a row:
1. filter people by sex
2. filter and use calculation if century parameter is present 

this approach will use calculations `Math.ceil(p.died / 100)` only on filtered data and not an all.
```suggestion
  const men = people.filter((p) => p.sex === 'm').filter((p) => century ? Math.ceil(p.died / 100) === century : true)
```